src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -1663,14 +1668,17 @@ private ImmutableArray<BoundExpression> BuildArgumentsForErrorRecovery(AnalyzedA
                         {
                             // bind the argument against each applicable parameter
                             var unboundArgument = (UnboundLambda)argument;
-                            foreach (var parameterList in parameterListList)
+                            if (!unboundArgument.HasBoundForErrorRecovery)
```

Comment:
>if (!unboundArgument.HasBoundForErrorRecovery) [](http://example.com/codeflow?start=28&length=46)

This check simply avoids binding the lambda for error recovery below when the binding will be ignored.